apps.json

```diff
@@ -3795,7 +3794,7 @@
         "maintained": false,
         "potential_alternative_to": [
             "BitTorrent",
-            "µTorrent®"
+            "\u00b5Torrent\u00ae"
```

Comment:
uuuh bro you fucked up the file encoding :|

Reply:
I just used the provided script. 
Revert the encoding